package java.exception.exceptionHandlingInJava.finallyBlock;

public class FinallyBlockExample {
    public static void main(String[] args) {
        try {
            String str = null;
            System.out.println(str.length());
        } catch (Exception e) {
            throw new RuntimeException(e);
        } finally {
            System.out.println("Finally block will always execute either any exception occur or not");
        }
    }
}
